contrib/grinder/scripts/utils.py

```diff
@@ -28,7 +28,7 @@
   'enum_num_values': 10,
   'singleplot_per_interval': 10,
   'annotations_queries_per_interval': 8,
-  'ingest_delay_millis': 0}
+  'ingest_delay_millis': ""}
```

Comment:
looks like you are now expecting a comma separated value? can you put comments here or the properties file the format? Is it always 3 values? What are the 1st, 2nd, 3rd values for, etc?